README.md

```diff
@@ -15,24 +15,29 @@ URL: https://www.nrel.gov/docs/fy17osti/67847.pdf
 ## Introduction
 
 bifacialvf is a self-contained view factor (or configuration factor)
-model which replicates a 5-row PV system of infinite extent perpendicular to the module rows. 
+model which replicates a single row of a PV system of infinite extent perpendicular to the module rows. 
```

Comment:
This isn't true.  The code assumes 5 rows of PV - the middle (interior) row is simulated by default, but user interface options include 'first, interior, last, single'.  We could add a statement that these are the simulation options..